docs/src/index.md

```diff
@@ -24,8 +24,11 @@ julia> using Pkg
 julia> Pkg.add("Oceananigans")
 ```
 
-!!! compat "Julia 1.9 or later is required; Julia 1.10 is suggested"
-    Oceananigans requires Julia 1.9 or later. We strongly urge you to use Julia 1.10.
+!!! compat "Julia 1.9 or later is required"
+    Oceananigans requires Julia 1.9 or later.
+    
+!!! info "Continuous integration testing"
+    Oceananigans is tested on Julia 1.10.
```

Comment:
```suggestion
    Oceananigans is currently tested on Julia 1.10.
```

Reply:
What does "currently" add here? The docs are associated with package versions; later on for version 0.423.2 we may change this to Julia 1.12. Right?

Reply:
My intent is that if someone notices it is wrong it gives them a clue why and they can submit a PR to fix it